src/test/java/org/openrewrite/java/testing/junit5/UpdateBeforeAfterAnnotationsTest.java

```diff
@@ -92,10 +119,34 @@ void after() {
                   }
               }
               """
+          ),
+          //language=kotlin
+          kotlin(
+            """
+              import org.junit.After
+
+              class Test {
+
+                  @After
+                  fun after() {
+                  }
+              }
+              """,
+            """
+              import org.junit.jupiter.api.AfterEach
+
+              class Test {
+
+                  @AfterEach
+                  fun after() {
+                  }
+              }
+              """
```

Comment:
For this class in particular we're only using the ChangeType recipes, which are extensively tested elsewhere, such that I think we can remove all but one of these Kotlin assertions from the tests.

Reply:
Done @timtebeek